app/src/main/java/com/nowak/wjw/simplecompass/domain/GetDestinationBearingUseCase.java

```diff
@@ -0,0 +1,43 @@
+package com.nowak.wjw.simplecompass.domain;
+
+import android.location.Location;
```

Comment:
Reviewer comment - "There’s no clear separation of architecture layer. You made a `domain` package, but inside are Android SDK dependencies."
I don't understand why Location cannot be here.
I need to take it and process it.

I can understand that it shouldn't be in viewModel or fragment,- but here? - why not?

Reply:
Osoba która robiła ci review zwraca ci uwagę na to że warstwa domeny posiada zależności stricte z androida.
Generalnie z tego co widzę to próbujesz użyć wszystkiego na co zwrócisz uwagę na internecie, a brakuję ci wiedzy i doświadczenia żeby to wszystko spiąć w logiczną całość. Nawet radziłbym używać mniej złożonej architektury a starać się oddzielić warstwę prezentacji,danych i logiki.